catalog/ui/catalog-ui-search/src/main/webapp/component/input/autocomplete/input-autocomplete.less

```diff
@@ -0,0 +1,35 @@
+@{customElementNamespace}input.is-autocomplete {
+    .select2-container--default .select2-selection--single {
+        background: @darken-inherited-background-color;
+        color: @button-text-color;
+        border: 1px solid @heavily-lighten-inherited-background-color;
+        border-radius: @border-radius;
+        height: auto;
+
+        .select2-selection__rendered {
+            line-height: @minimumButtonSize;
+            padding-right: @minimumButtonSize;
+        }
+
+        .select2-selection__arrow {
+            top: 0px;
+            height: 100%;
+            width: @minimumButtonSize;
+        }        
+    }
+
+
+}
+
+// can't namespace because gets appended to doc
```

Comment:
I think it'd be worth adding:
```
.select2-results__option {
    height: @minimumButtonSize;
    line-height: @minimumButtonSize;
    overflow: hidden;
    text-overflow: ellipsis;
    padding: 0px @minimumSpacing;
    white-space: nowrap;
}
```
This will bring it more in line with the other dropdowns and handle text overflow.